tagstudio/tests/src/core/test_tags.py

```diff
@@ -1,12 +1,14 @@
-from src.core.library import Tag
+import unittest
 
+from tagstudio.src.core.library import Tag
```

Comment:
`Tagstudio.src...` import format breaks at runtime for vscode and when running from the terminal

```suggestion
from src.core.library import Tag
```

Reply:
Thank you for the feedback! I have made the changes.